nservicebus/dependency-injection/nservicebus-dependencyinjection.md

```diff
@@ -0,0 +1,50 @@
+---
+title: NServiceBus.Extensions.DependencyInjection
+summary: Provides integration with the Microsoft.Extensions.DependencyInjection abstraction.
+reviewed: 2020-02-17
+component: Extensions.DependencyInjection
+tags:
+ - Dependency Injection
+related:
+ - samples/dependency-injection/extensions.dependencyinjection
+---
+
+The `NServiceBus.Extensions.DependencyInjection` package provides integration with the `Microsoft.Extensions.DependencyInjection` dependency injection abstraction.
+
+
+## Usage with ServiceCollection
+
+The following snippet shows how to configure NServiceBus to use Microsoft's built-in dependency injection container:
+
+snippet: usecontainer-servicecollection
+
+
+## Usage with third party containers
+
+NServiceBus can also be configured to work with any third party dependency injection which implements the `Microsoft.Extensions.DependencyInjection` abstraction. To use a third party dependency injection container, pass the specfic container's `IServiceProviderFactory` to the `UseContainer` configuration method. The following snippet shows this approach, using Autofac (`Autofac.Extensions.DependencyInjection`) as an example:
```

Comment:
```suggestion
NServiceBus can also be configured to work with any third party dependency injection container which implements the `Microsoft.Extensions.DependencyInjection` abstraction. To use a third-party dependency injection container, pass the specific container's `IServiceProviderFactory` to the `UseContainer` configuration method. The following snippet shows this approach, using Autofac (`Autofac.Extensions.DependencyInjection`) as an example:
```

Reply:
I like those suggestions, but you're using third-party and third party, should align those cases?